### PR TITLE
Clarify how to improve a core library

### PR DESCRIPTION
**Pull request by goldfirere**
Based on a recent thread, there may be difficulty in knowing how to proceed with a core-library suggestion. I'm hoping this clarifies. Are the procedures here correct?

To be clear, I'm *not* trying to recommend new procedures and/or change anything about the functioning of the CLC. I am just trying to clarify and be explicit about what I believe is already the state of affairs, and I thought submitting this PR would be helpful in this regard.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,3 +58,11 @@ may apply for adoption by CLC.
 Core Libraries are different from GHC boot libraries: some boot libraries are not Core
 (and as such are maintained either solely by GHC Team, or by third parties), and some
 Core Libraries are not required to boot GHC.
+
+### Suggesting changes to Core libraries
+
+Because the CLC owns these libraries but does not maintain them, if you have an improvement
+to suggest for a core library (including one that affects several), please start by containing
```
Comment (cigsender) [inline README.md:65]:
```suggestion
to suggest for a core library (including one that affects several), please start by contacting
```

**Review by cigsender — APPROVED** [review]
Thanks Richard, this is an important step that we neglected to document.

**Review by Bodigrim — APPROVED** [review]
Thanks @goldfirere!

**Review by emilypi — APPROVED** [review]

**Comment by Bodigrim** [comment]
Since this is not a change of policy, but rather a mere clarification of the paragraph above, merging.

**Pull request merged**